examples/notebooks/Poisson_non_periodic.ipynb

```diff
@@ -1,6 +1,7 @@
 {
  "cells": [
   {
+   "attachments": {},
```

Comment:
I don't see the value in updating this Jupiter Notebook file, with the exception of the lines that I point out below.

Reply:
Agreed. I can undo the change. These lines were part of the "hidden" output, I did not change them on purpose.

Reply:
I know. After modifying a Jupiter notebook one often has to stage the changes line by line using the `-p` flag:

```bash
git add -p filename.ipynb
```